framework/patch_dispatch_change_test.go

```diff
@@ -0,0 +1,213 @@
+package framework
+
+import (
+	"context"
+	"fmt"
+	"testing"
+)
+
+func Test_Framework_Resource_PatchDispatchChange(t *testing.T) {
```

Comment:
This should make sure dispatching the patch changes work as expected. We did not seem to have such tests so far.